components/config/src/config.rs

```diff
@@ -152,6 +152,8 @@ pub struct Config {
     #[serde(skip_serializing, skip_deserializing)] // not a typo, 2 are need
     pub extra_syntax_set: Option<SyntaxSet>,
 
+    pub link_checker_skip_anchor_prefixes: Vec<String>,
```

Comment:
Sorry, I meant having the struct

```rust
#[derive(Clone, Debug, PartialEq)]
pub struct LinkChecker {
    /// Skip anchor checking for these URL prefixes
    pub skip_anchor_prefixes: Vec<String>,
}
```
in this file and use it in the config

Reply:
Oh, I misunderstood your wording then.